calico-cloud_versioned_docs/version-3.16/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -1,40 +1,47 @@
 ---
-description: Scan images using CLI scanner.
+description: Integrate the Image Assurance scanner into your build pipeline.
 ---
 
-# Scan images using the CLI scanner
+# Integrate Image Assurance scanner in your build pipeline
 
 ## Big picture
 
-Scan images manually or via CI/CD pipeline for vulnerabilities using {{prodname}} Image Assurance CLI scanner.
+Integrate the Image Assurance scanner into your build pipeline and private registries.
 
 ## Before you begin
 
 **Image requirements**
 
-To be included in scan results, images must:
+Images must:
 
-- Be available locally through the container runtime environment where the CLI scanner is running
+- Be available locally through the container runtime environment where the scanner is running
 - Have a RepoDigest ID
 
   To verify RepoDigestIDs, run this command:
 
   `docker images --digests`
```

Comment:
Deleted.

Reply:
Note that I added a eference to "containerized version" in the intro in the CLI scanner doc. Let me know if this can also be included when we release this doc. Or if we need a separate PR.